static/app/views/dashboardsV2/widgetCard/chart.tsx

```diff
@@ -169,7 +169,12 @@ class WidgetCardChart extends Component<WidgetCardChartProps, State> {
           data={result.data}
           organization={organization}
           stickyHeaders
-          getCustomFieldRenderer={(field, meta) => getFieldRenderer(field, meta, isAlias)}
+          getCustomFieldRenderer={
+            datasetConfig.getCustomFieldRenderer
+              ? (field, meta) =>
+                  datasetConfig.getCustomFieldRenderer!(field, meta, {organization})
```

Comment:
It complains without `!` because it could be undefined, but line 173 asserts that it's truthy 🤔 

I'm thinking either we make this required or leave it like this.

Reply:
ugh we need a way to deal with this